File: go/core/bizzfuzz-linio.go
package core

import "fmt"
import u "go-linio/util"
import "strconv"

/**
 * [LOOP_LIMIT how many times to iterate]
 * @type {int}
 */
const LOOP_LIMIT int = 100

/**
 * [string stores all posible outputs by case]
 * @type {[]string}
 */
var messages [3]string = [3]string{"IT", "Linio", "Linianos"}

/**
 * [isDivisibleByThree]
 * @type {int}
 */
var isDivisibleByThree uint8 = 0

/**
 * [isDivisibileByBoth]
 * @type {int}
 */
var isDivisibleByBoth uint8 = 0

/**
 * [FizzBuzzLinio function used to run linio challenge algorithm]
 */
func FizzBuzzLinio(){
	for i:=1; i<=LOOP_LIMIT; i++{
		isDivisibleByBoth 	= u.BoolToInt(i%3==0 && i%5==0)
		isDivisibleByThree 	= u.BoolToInt(i%3==0 && i%5>0)

		if i%3==0 || i%5==0{
			fmt.Println(messages[(isDivisibleByBoth*2)+isDivisibleByThree])
			continue
		}

		fmt.Println(i)
	}
}

/**
 * [GetMessageByNumber retrieves string from one number if is in case]
 * @param number 	int
 * @returns string
 */
func GetMessageByNumber(n int)string{
	if(n%3==0 || n%5==0){
		return messages[(u.BoolToInt(n%3==0 && n%5==0)*2)+u.BoolToInt(n%3==0 && n%5>0)]
	}

	return strconv.Itoa(n);
}

File: php/core/bizzfuzz-linio.php
<?php
	declare(strict_types=1);

	namespace Linio\Challenge;

	/**
	 * @class 	FizzBuzzLinio
	 * @desc 	Contains challenge solved algorithm
	 */
	final class FizzBuzzLinio{
		const MESSAGES 	= ["IT", "Linio", "Linianos"];
		const RUN_UNTIL = 100;

		/**
		 * [run starts the algorithm]
		 */
		public static function run():void{
			$is_divisible_by_both;
			$is_divisible_by_three;

			for($i=1;$i<=self::RUN_UNTIL;$i++){
				$is_divisible_by_both = (int)($i%3==0 && $i%5==0);
				$is_divisible_by_three = (int)($i%3==0 && $i%5>0);

				if($i%3==0 || $i%5==0){
					printf("%s \n", self::MESSAGES[($is_divisible_by_both*2)+$is_divisible_by_three]);
					continue;
				}

				printf("%d \n", $i);
			}
		}

		/**
		 * [getMessageByNumber select the message with given number following problem constraints]
		 * @param  int    $n
		 * @return string
		 */
		public static function getMessageByNumber(int $n):string{
			return ($n%5==0 || $n%3==0)?self::MESSAGES[(((int)($n%3==0 && $n%5==0))*2)+((int)($n%3==0 && $n%5>0))]:((string)$n);
		}
	}